tests/ignite/contrib/metrics/regression/test_median_absolute_percentage_error.py

```diff
@@ -135,7 +135,7 @@ def _test(metric_device):
         np_res = 100.0 * np.median(e)
 
         e_prepend = np.insert(e, 0, e[0], axis=0)
-        np_res_prepend = 100.0 * np.median(e)
```

Comment:
I didn't notice that if we now compute the result as torch.median, no need to match between two results ?

Reply:
Yes we can do that, but the assertion here is between `res (computed by ignite)` and `np_res (computed by numpy)`, and that's what are we doing now (comparing between value computed by torch and value computed by numpy)

Reply:
>>> no need to match between two results ?
Can you explain that more?

Reply:
Do we need this : 
```
assert pytest.approx(res) in (np_res, np_res_prepend)
# vs
# assert pytest.approx(res) in np_res_prepend
```

Reply:
Well, i guess using the matching between two results is good to show that this `np_res_prepend` is a special case, what do you think?

Reply:
In this case, I'd do if something then check with this result otherwise check with another one.

Reply:
Something like this?
```
if idist.get_world_size() % 2 == 0:
        assert pytest.approx(res) == np_res_prepend
else:
        assert pytest.approx(res) == np_res
```

Reply:
Once again, ws is almost always even :) Second is better.